ansible/roles/operator-pipeline/templates/openshift/tasks/show-support-link-with-pr.yml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: show-support-link-with-pr
```

Comment:
This task doesn't really seem to do anything with the PR at all, the posting is done in another task. So the name should probably be updated.

Reply:
Updated!